packages/core/package.json

```diff
@@ -5,7 +5,7 @@
     "access": "public"
   },
   "scripts": {
-    "dev": "vite dev",
+    "start": "vite dev",
```

Comment:
Do we use `start` or `dev` in other packages. I don't have a big preference since it's `pnpm x` either way, but we should pick and stick to a convention.

Reply:
I left `dev` for `core` but `legacy` uses `start`. I prefer `dev` because it implies that this is not used outside of development, which it is not.